crml/cennzx-spot/rpc/src/lib.rs

```diff
@@ -42,6 +42,14 @@ pub trait CennzxSpotApi<AssetId, Balance> {
 	#[rpc(name = "cennzx_sellPrice")]
 	// TODO: change to Result<Balance> once https://github.com/serde-rs/serde/pull/1679 is merged
 	fn sell_price(&self, asset_to_sell: AssetId, amount_to_buy: Balance, asset_to_payout: AssetId) -> Result<u64>;
+
+	#[rpc(name = "cennzx_liquidityValue")]
+	// TODO: change to Result<Balance> once https://github.com/serde-rs/serde/pull/1679 is merged
```

Comment:
Do we have a jira ticket for TODOs?

Reply:
^ Eric put something together to address this a while ago - but I will capture it in a ticket =)